draft-ietf-core-href.md

```diff
@@ -555,7 +555,17 @@ references:
 * An empty outer array (`[]`) is not a valid CRI.
   It is a valid CRI reference,
   equivalent to `[0]` as per {{ingest}}, which essentially copies the
-  base CRI.
+  base CRI up to and including the path section, unsetting query and fragment.
+
+| Section   | Default Value |
+|-----------|---------------|
+| scheme    | –             |
+| authority | null          |
+| discard   | –             |
```

Comment:
Given the line above on the empty array, isn't it 

```suggestion
| discard   | 0             |
```

because that is the value that gets removed from the tail?